### PR TITLE
Fixed a bunch of warnings during build (javadoc, lombok, generics)

### PR DESCRIPTION
**Pull request by psjobergprivat**

### CODE REVIEW

**Comment by psjobergprivat** [comment]
test me

File: service/src/main/java/eu/europa/ec/fisheries/ers/fa/dao/FishingActivityDao.java [hunk 1]
```diff
@@ -108,7 +108,6 @@ public int getPreviousFishingActivityId(int fishingActivityId, String activityTy
      * @param fishingActivityId
      * @param activityTypeCode
      * @param activityCalculatedStartTime
-     * @return
```
Comment (robertkrantz) [inline service/src/main/java/eu/europa/ec/fisheries/ers/fa/dao/FishingActivityDao.java:111]: Why remove the return? Wouldn't it be better to add some documentation? Either we try to keep it up to date or we remove it completely.

**Review by robertkrantz — APPROVED** [review]
Only comment I have is the one on the javadoc returns being removed.

File: service/src/main/java/eu/europa/ec/fisheries/ers/fa/dao/FishingActivityDao.java (hunk 1 above)
Comment (psjobergprivat) [reply service/src/main/java/eu/europa/ec/fisheries/ers/fa/dao/FishingActivityDao.java:111]: If there is relevant info that is not expressed via method/parameter names&types I think it is great if that was added as javadoc. This is out of scope for this commit which was just about getting rid of build warnings without loosing any functionality or information

**Pull request merged**